app/src/main/java/com/guzelgimadieva/tasky/authorization/ui/register/RegisterScreen.kt

```diff
@@ -0,0 +1,188 @@
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.guzelgimadieva.tasky.R
+import com.guzelgimadieva.tasky.authorization.ui.components.InputTextField
+
+@Composable
+fun RegisterScreen(
+    onBackClick: () -> Unit,
+) {
+    val viewModel: RegisterScreenViewModel = hiltViewModel()
+    val registerState by viewModel.registerState.collectAsState()
+    RegisterScreenContent(
+        onAction = viewModel::onEvent,
+        registerState = registerState,
+        onBackClick = onBackClick,
+    )
+}
+@Composable
+fun RegisterScreenContent (
+    onAction: (RegisterEvent) -> Unit,
+    registerState: RegisterScreenState,
+    onBackClick: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
+            .windowInsetsPadding(WindowInsets.statusBars),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = stringResource(id = R.string.welcome_screen_title),
+            style = MaterialTheme.typography.headlineLarge,
+            fontWeight = FontWeight.Bold,
+            color = Color.White,
+            modifier = Modifier
+                .padding(
+                    vertical = 64.dp,
+                )
+        )
+
+        Card(
+            modifier = Modifier
+                .fillMaxSize(),
+            shape = RoundedCornerShape(
+                topStart = 30.dp,
+                topEnd = 30.dp,
+            ),
+            colors = CardDefaults.cardColors(
+                containerColor = Color.White,
+            ),
+        ) {
+            Column(
+                modifier = Modifier
+                    .padding(16.dp)
+                    .fillMaxWidth(),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                InputTextField(
+                    value = registerState.username,
+                    onValueChange = { onAction(RegisterEvent.EmailChanged(it)) },
+                    placeholder = {
+                        Text(
+                            text = stringResource(id = R.string.register_screen_name_input),
+                            style = TextStyle(color = Color.Black.copy(alpha = 0.5f))
```

Comment:
I'd recommend to use `LocalTextStyle.current.copy(...)` instead as that considers the text style created in your material theme. Creating a completely new text style with `TextStyle(...)` won't do that and will therefore not use your project font, etc.